modules/storage-expanding-recovering-failure.adoc

```diff
@@ -7,13 +7,21 @@
 [id="expanding-recovering-from-failure_{context}"]
 = Recovering from failure when expanding volumes
 
-If expanding underlying storage fails, the {product-title} administrator can manually recover the persistent volume claim (PVC) state and cancel the resize requests. Otherwise, the resize requests are continuously retried by the controller.
+If a resize request fails or remains in a pending state, you can try again by entering another smaller resize value in `.spec.resources.requests.storage` for the persistent volume claim (PVC). The smaller value should be smaller than the original volume size.
```

Comment:
The smaller value should NOT be smaller than the original volume size.

Reply:
Oh dear. Yes! Fixed.